components/automate-cli/cmd/chef-automate/applications.go

```diff
@@ -168,7 +171,14 @@ func newApplicationsRemoveSvcsCmd() *cobra.Command {
 	c := &cobra.Command{
 		Use:   "remove-svcs",
 		Short: "Remove services from the applications database",
-		RunE:  runApplicationsRemoveSvcsCmd,
+		Long: `
+Remove services from the applications database.
+
+You must decommission the services prior to running this command. Services
+removed with this command will be added to the database again if Automate
+receives a health check from the service.
```

Comment:
Asking for clarification. Does "Services removed with this command will be added to the database again if Automate receives a health check from the service" mean "Unless a service is fully decommissioned before it is removed with this command, it will send a health-check at the appointed time, at which point Automate will re-add it to the services (nodes?) database"?

Reply:
> Unless a service is fully decommissioned before it is removed with this command, it will send a health-check at the appointed time, at which point Automate will re-add it to the services (nodes?) database

Yes, exactly this.